catalyst/utils/tools/metric_manager.py

```diff
@@ -141,4 +112,4 @@ def add_batch_value(
             self._batch_values[metric_name] = metric_value
```

Comment:
can you add `add_epoch_value` similar to `add_batch_value` for only epoch metrics?

Reply:
MetricManager is deprecated now... finally
and have no usages in the whole framework